imager/imager/templates/profile.html

```diff
@@ -0,0 +1,40 @@
+{% extends 'base.html' %}
+{% load staticfiles %}
+{% load thumbnail %}
+{% block content %}
+
+<div id='profile_content'>
+    <h1>Welcome, {% if user.first_name %}{{user.first_name}}{% else %}{{user}}{% endif %}!</h1>
```

Comment:
might be nice to have a method on the profile that does this, so you can just say `{{user.profile.name}}`